config/php-fpm/docker-php-ext-xdebug.ini

```diff
@@ -1 +1,6 @@
 zend_extension=/usr/local/lib/php/extensions/no-debug-non-zts-20151012/xdebug.so
+
+[Xdebug]
+xdebug.remote_enable = 1
+xdebug.remote_log = /var/log/xdebug/xdebug.log
+xdebug.remote_host = 172.18.0.1
```

Comment:
`172.18.0.1` is the address defined on `docker-compose.yml`. It works on any linux install - untested on Macs but it should work too. Does not work on Windows installs (see note above and on the readme.md).